app/(afterLogin)/dashboard/_components/approval-status/index.tsx

```
@@ -6,6 +6,7 @@
 import { useGetApprovalLeaveRequest } from '@/store/server/features/timesheet/leaveRequest/queries';
 import { useDashboardApprovalStore } from '@/store/uistate/features/dashboard/approval';
 import { useGetBranchTransferApproveById } from '@/store/server/features/employees/approval/queries';
+import { APPROVALTYPES } from '@/types/enumTypes';
```

ESLint

[error] 'APPROVALTYPES' is defined but never used.

(@typescript-eslint/no-unused-vars)